addons/mail/static/src/scss/variables/primary_variables.scss

```diff
@@ -1,4 +1,4 @@
-$o-mail-thread-avatar-size: 36px !default;
+$o-mail-thread-avatar-size: $o-avatar-size * .4 !default;
```

Comment:
Is there a specific reason or do we have a guideline to omit the 0 before decimal? I find it harder to read, it looks a bit like a typo and at glance when scanning code it's easier to miss the lonely dot and think it's 4.